source/docs/api-reference/api-usage/index.rst

```diff
@@ -30,14 +30,43 @@ There are three major components to the Phoenix Pro API:
 
       Signals represent **data retrieved** from a device. This can be velocity, position, yaw, pitch, roll, temperature, etc.
 
-.. mimictoc::
+- :doc:`api-overview`
+   - Details a high level overview of what makes up the Phoenix Pro API.
+
+- :doc:`configuration`
+   - Describes configuring device configs via code.
+
+- :doc:`control-requests`
+   - Highlights using control requests to control the open and closed loop functionality of actuators such as the TalonFX.
+
+- :doc:`status-signals`
+   - Details using status signals to retrieve sensor data from devices.
+
+- :doc:`faults`
+   - Documents how faults are used to indicate device hardware status.
+
+- :doc:`enabling-actuators`
+   - Information on the FRC Lock safety feature and how to feed devices.
+
+- :doc:`actuator-limits`
+   - Documents how to use stator/supply current limits and limit switches.
```

Comment:
Replace with:
Documents how to retrieve and configure software and hardware actuator limits.